http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala

```diff
@@ -488,7 +492,7 @@ class AKSInterpreter[F[_]](config: AKSInterpreterConfig,
         raw"relaylistener.runtimeName=${appName.value}",
         raw"relaylistener.image=${config.listenerImage}",
         raw"""relaylistener.removeEntityPathFromHttpUrl="${removeEntityPathFromHttpUrl.toString}"""",
-
+        raw"""relaylistener.validHosts=[${validHosts.mkString("','")}]'""",
```

Comment:
Currently unsure the syntax the values file needs for this... searching for examples.

Reply:
Maybe just
```
raw"""relaylistener.validHosts=${validHosts.mkString(",")}""",
```
?

So you're not really passing it as an array, but a string to helm

Reply:
I'll give it a shot